integration-tests/smoke/keeper_test.go

```diff
@@ -380,12 +385,13 @@ var _ = Describe("Keeper Suite @keeper", func() {
 			log.Info().Int64("Upkeep counter", existingCnt.Int64()).Msg("Upkeep counter when check gas increased")
 
 			// In most cases count should remain constant, but there might be a straggling perform tx which
-			// gets committed later. Hence, we check that the upkeep count does not increase by more than 1
+			// gets committed later. Since every keeper node cannot have more than 1 straggling tx, it
+			// is sufficient to check that the upkeep count does not increase by more than 6.
 			Consistently(func(g Gomega) {
 				cnt, err := consumerPerformance.GetUpkeepCount(context.Background())
 				g.Expect(err).ShouldNot(HaveOccurred(), "Calling consumer's counter shouldn't fail")
 				g.Expect(cnt.Int64()).Should(
-					BeNumerically("<=", existingCnt.Int64()+1),
+					BeNumerically("<=", existingCnt.Int64()+6),
```

Comment:
thank you for fixing this. Could you also fix line 411 to have 6 instead of 1? Maybe create a constant variable to use in multiple places

Reply:
No problem!! Sure thing, will create the constant in the next commit.